### PR TITLE
Telegram notifications + replies.

### PR DESCRIPTION
**Pull request by ice-myles**
Telegram notifications + replies.

Depends on https://github.com/ice-blockchain/wintr/pull/48

### CODE REVIEW

File: notifications/DDL.sql [hunk 1]
```diff
@@ -16,8 +16,12 @@ CREATE TABLE IF NOT EXISTS users  (
                     profile_picture_name                    TEXT,
                     referred_by                             TEXT,
                     phone_number_hash                       TEXT,
+                    telegram_user_id                        TEXT NOT NULL,
+                    telegram_bot_id                         TEXT NOT NULL,
                     language                                TEXT NOT NULL default 'en'
                   );
+ALTER TABLE users ADD COLUMN IF NOT EXISTS telegram_user_id text NOT NULL;
+ALTER TABLE users ADD COLUMN IF NOT EXISTS telegram_bot_id text NOT NULL;
```
Comment (ice-ares) [inline notifications/DDL.sql]: Not gonna work. U need a func that sets it atomically.

1. Add nullable column if not exists
2. Update column, set value = user_id
3. Alter table make column not null


U have examples in eskimo/santa/husky how to do that

File: notifications/DDL.sql [hunk 2]
```diff
@@ -45,7 +49,7 @@ CREATE TABLE IF NOT EXISTS scheduled_notifications  (
                     notification_channel_value  TEXT NOT NULL,
                     primary key(user_id,uniqueness,notification_type,notification_channel,notification_channel_value));
 CREATE UNIQUE INDEX IF NOT EXISTS scheduled_notifications_i_ix ON scheduled_notifications (i);
-CREATE INDEX IF NOT EXISTS scheduled_notifications_mod_i_ix ON scheduled_notifications (MOD(i, %[1]v), scheduled_for ASC);
+CREATE INDEX IF NOT EXISTS scheduled_notifications_mod_i_ix ON scheduled_notifications (MOD(i, %[1]v), scheduled_for, notification_channel ASC);
```
Comment (ice-ares) [inline notifications/DDL.sql]: This is not gonna do anything. U need to replace the index

File: notifications/translations/push/mining_expired.txt [hunk 3]
```diff
@@ -1,202 +1,202 @@
 {
     "en": {
-        "title": "⏰  Tick-tock, tick-tock!",
-        "body": "Keep your coins safe by starting a new check-in session now."
```
Comment (ice-ares) [inline notifications/translations/push/mining_expired.txt:4]: Dont u need text per tenant?

File: notifications/DDL.sql (hunk 1 above)
Comment (ice-ares) [reply notifications/DDL.sql]: Or atleast tell me the exact sql to run

File: notifications/translations/push/mining_expired.txt (hunk 3 above)
Comment (ice-myles) [reply notifications/translations/push/mining_expired.txt:4]: We did it for kyc only. And here more common translations. At least we didn't use it for fcm pushes.

File: notifications/DDL.sql (hunk 1 above)
Comment (ice-ares) [reply notifications/DDL.sql]: Actually no, lets make both columns nullable. And just handle the select with a coalesce

**Pull request merged**